mlir/include/mlir/Dialect/MIOpen/MIOpen.h

```diff
@@ -111,8 +111,8 @@ class CoordTransformsBuilder {
   // The names live as long as the CoordTransformBuilder
   void getEndNames(SmallVectorImpl<StringRef> &names);
 
-  SmallString<8> startName(uint32_t dim);
-  SmallString<8> endName(uint32_t dim);
+  StringRef startName(uint32_t dim);
```

Comment:
No, that's not acceptable, since we can't guarantee the lifetime of the returned string within the builder.